configs/config.clevo_ns70pu

```diff
@@ -0,0 +1,31 @@
+CONFIG_CCACHE=y
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_NO_STAGE_CACHE=y
+CONFIG_VENDOR_CLEVO=y
+CONFIG_MAINBOARD_VERSION="darp8"
```

Comment:
This can be read from the motherboard. On TGL-U all models we support had versions v2.2a.

Reply:
Changed to 'v2.1